src/patterns/service-unavailable-pages/default.njk

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout-example.njk
+---
+
+{% from "panel/macro.njk" import govukPanel %}
```

Comment:
This line can be removed – it is importing the panel component, but the panel isn't being used as part of this example.